api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -8,52 +8,61 @@
 ENVIRONMENTS = {
   "all-of-us-workbench-test" => {
     :publisher_account => "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com",
-    :source_cdr_project => "all-of-us-ehr-dev",
-    :ingest_cdr_project => "fc-aou-vpc-ingest-test",
-    :dest_cdr_project => "fc-aou-cdr-synth-test",
-    :config_json => "config_test.json"
+    :accessTiers => {
+      "registered" => {
+        :source_cdr_project => "all-of-us-ehr-dev",
+        :ingest_cdr_project => "fc-aou-vpc-ingest-test",
+        :dest_cdr_project => "fc-aou-cdr-synth-test",
+        :auth_domain_group_email => "GROUP_all-of-us-registered-test@dev.test.firecloud.org",
```

Comment:
PRO: This file contains all the config values it needs
CON: auth_domain_group_email is duplicated in another config file

Alternatively we could update get_auth_domain_group_email() to reference the new config file